engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/helpers/ProcessTestHelpers.scala

```diff
@@ -90,18 +111,19 @@ object ProcessTestHelpers {
         ExpressionConfig(globalProcessVariables, List.empty, dictionaries = Map(dictId -> WithCategories(dictDef)))
       }
 
-      override def signals(config: Config): Map[String, WithCategories[ProcessSignalSender]] = Map()
+      override def signals(config: Config): Map[String, WithCategories[TestProcessSignalFactory]] = Map("sig1" ->
+        WithCategories(new TestProcessSignalFactory(kafkaConfig, signalTopic)))
 
       override def buildInfo(): Map[String, String] = Map.empty
     }
 
     def invokeWithKafka(process: EspProcess, kafkaConfig: KafkaConfig,
                         config: Configuration = new Configuration(),
                         processVersion: ProcessVersion = ProcessVersion.empty,
-                        parallelism: Int = 1): JobExecutionResult = {
+                        parallelism: Int = 1)(action: => Unit): Unit = {
```

Comment:
Can you name this `action` more explicitly? It is kind of `postRunAction`?

Reply:
I change names to more informative